doc/src/devdocs/callconv.md

```diff
@@ -1,60 +1,43 @@
-.. _devdocs-callconv:
-
-*******************
-Calling Conventions
-*******************
-
-.. currentmodule:: Base
+# Calling Conventions
 
 Julia uses three calling conventions for four distinct purposes:
 
-+-------+--------------+----------------------------------+
-|Name   |Prefix        |Purpose                           |
-+=======+==============+==================================+
-|Native |``julia_``    | Speed via specialized signatures |
-+-------+--------------+----------------------------------+
-|JL Call|``jlcall_``   | Wrapper for generic calls        |
-+       +--------------+----------------------------------+
-|       |``jl_``       | Builtins                         |
-+-------+--------------+----------------------------------+
-|C ABI  |``jlcapi_``   | Wrapper callable from C          |
-+-------+--------------+----------------------------------+
+| Name    | Prefix    | Purpose                          |
+|:------- |:--------- |:-------------------------------- |
+| Native  | `julia_`  | Speed via specialized signatures |
+| JL Call | `jlcall_` | Wrapper for generic calls        |
+| JL Call | `jl_`     | Builtins                         |
```

Comment:
does markdown not have a way of making table entries extend across multiple rows?